trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.js

```diff
@@ -79,4 +79,16 @@ describe(page, () => {
 
     wrapper.expectElement(`${premiumPaymentsTabSelector} table[data-cy="premium-payments-table"]`).toExist();
   });
+
+  it('should render the facility ID filter input', () => {
+    wrapper.expectElement('[data-cy="facility-filter-input"]').toExist();
+  });
+
+  it('should render the facility ID filter submit button', () => {
+    wrapper.expectElement('[data-cy="facility-filter-submit-button"]').toExist();
+  });
+
+  it('should render the facility ID filter clear button', () => {
+    wrapper.expectElement('[data-cy="facility-filter-clear-button"]').toExist();
+  });
```

Comment:
I'm not a big fan of tests that just test for existence of a data-cy, fine to select elements using data-cy as long as we are then using them for something or asserting their content etc, e.g. could get the filter input by data-cy and then validate that the value is set to the passed in query param/not initiated when there is no param.

I think also the `params` above actually need updating as the view model type has changed, and I think we could also possibly add some extra tests around the errors now there is more logic happening there inside the template itself.